zulipterminal/ui_tools/boxes.py

```diff
@@ -208,6 +208,37 @@ def update_recipient_emails(self, write_box: ReadlineEdit) -> None:
             write_box.edit_text
         )
 
+    def tidy_and_validate_recipients(self,
+                                     write_box: ReadlineEdit) -> List[str]:
+        tidied_recipients = []
+        invalid_recipients = []
+        recipients = [recipient.strip()
+                      for recipient in write_box.edit_text.split(',')
+                      if recipient.strip()]
+
+        for recipient in recipients:
+            cleaned_recipient_list = re.findall(
+                r"^(.*?)(?:\s*?<?([\w\.-]+@[\w\.-]+)>?(.*))?$",
+                recipient
+            )
+            recipient_name, recipient_email, invalid_text = (
```

Comment:
minor: If we aren't interested in `invalid_text`, we could replace it with `_` :)

Reply:
We're not interested in `invalid_text` but I'm inclined to let the name stay so that it's somewhat understandable from reading the code why we're discarding it, but that also sounds like a need for some comments within, I'll add them.